content/fr/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,344 @@
+
+```bash
+kubectl logs my-pod                                 # Affiche les logs du pod (stdout)
+kubectl logs my-pod --previous                      # Affiche les logs du pod (stdout) pour une instance précédente du container
+kubectl logs my-pod -c my-container                 # Affiche les logs d'un container particulier du pod (stdout, cas d'un pod multi-container)
+kubectl logs my-pod -c my-container --previous      # Affiche les logs d'un container particulier du pod (stdout, cas d'un pod multi-container) pour une instance précédente du container
+kubectl logs -f my-pod                              # Fait défiler (stream) les logs du pod (stdout)
+kubectl logs -f my-pod -c my-container              # Fait défiler (stream) les logs d'un container particulier du pod (stdout, cas d'un pod multi-container)
+kubectl run -i --tty busybox --image=busybox -- sh  # Exécute un pod comme un shell interactif
+kubectl attach my-pod -i                            # Attache à un container en cours d'exécution
+kubectl port-forward my-pod 5000:6000               # Écoute le port 5000 de la machine locale et forwarde vers le port 6000 de my-pod
+kubectl exec my-pod -- ls /                         # Exécute une commande dans un pod existant (cas d'un seul container)
+kubectl exec my-pod -c my-container -- ls /         # Exécute une commande dans un pod existant (cas multi-container)
+kubectl top pod POD_NAME --containers               # Affiche les métriques pour un pod donné et ses containers
+```
+
+## Interaction avec des Noeuds et Clusters
+
+```bash
+kubectl cordon mon-noeud                                              # Marque mon-noeud comme non assignable (unschedulable)
+kubectl drain mon-noeud                                               # Draine mon-noeud en préparation d'une mise en maintenance
+kubectl uncordon mon-noeud                                            # Marque mon-noeud comme assignable
+kubectl top node mon-noeud                                            # Affiche les métriques pour un noeud donné
+kubectl cluster-info                                                  # Affiche les adresses du master et des services
+kubectl cluster-info dump                                             # Affiche l'état courant du cluster sur stdout
+kubectl cluster-info dump --output-directory=/path/to/cluster-state   # Affiche l'état courant du cluster sur /path/to/cluster-state
+
+# Si une teinte avec cette clé et cet effet existe déjà, sa valeur est remplacée comme spécifié.
+kubectl taint nodes foo dedicated=special-user:NoSchedule
+```
+
+### Types de ressources
+
+Liste tous les types de ressources pris en charge avec leurs noms courts (shortnames), [groupe d'API (API group)](/docs/concepts/overview/kubernetes-api/#api-groups), s'ils sont [cantonnés à un espace de nom (namespaced)](/docs/concepts/overview/working-with-objects/namespaces), et leur [Genre (Kind)](/docs/concepts/overview/working-with-objects/kubernetes-objects):
```

Comment:
suggest "s'ils sont" -> elles sont.
should we translate espace de nom? too weird imho...